### ISSUE TITLE
Guide to use?

### ISSUE THREAD

**Issue by nickcartery** [rdkcentral/OCDM-Playready#16]
Can i use this repo with ffmpeg to decrypt PlayReady stream? or Can this one extract the key (16bit) ?

**Comment by 3052** [comment]
can someone comment? @HaseenaSainul  @MFransen69 @bramoosterhuis

**Comment by pwielders** [comment]
This is a wrapper around the Microsoft playready library and exposes an interface that hooks into ocdm client (ThunderClientLibraries). The ocdm client library is used by all native apps (Netflix/webkit/amazon and Disney) to receive the acquisition license (from this module), feedback the response (to this module) and potentially feed the decryption keys into the SVP pipeline (if applicable) and feed a reference into the gstreamer pipeline. 

Bottom line this is an adapter for the Microsoft Playready library (you can obtain that from Microsoft if you are licensed) to enable the OCDM framework (part of ThunderPlugin OCDM) to move licenses and keys around do that native apps can do decrypted playback. 

There is, for example, a similar wrapper for Widevine, exposing the same interface but than used by Cobalt App (YouTube). 

So this will not fly with ffmpeg.

**Comment by 3052** [comment]
> So this will not fly with ffmpeg.

not really true. FFmpeg will decrypt media if you give it a key. the problem is FFmpeg has no interaction with a license server, so that part needs to be handled by a separate tool
